accepted/00060-hub-general.md

```diff
@@ -0,0 +1,109 @@
+    * Packaged software
+    * Configuration Management data (states, pillars...)
+    * OS Images
+    * Container Images
+  * Managed content can be selectively synced from Hub to Servers
+    * Same order of importance defined above
+    * Synchronization has to be atomic
+    * Synchronization has to be doable via a storage media instead of a network for certain network-constrained environments (defense, China)
+    * Nice-to-have: avoid the download of content from the Hub if it is faster to retrieve it from SCC
+    * Nice-to-have: on-demand downloading from the Hub should be considered (Proxy/Squid model)
+  * [Uyuni for Retail extension](https://github.com/uyuni-project/retail) is updated not to require a Proxy
+  * Administration of Servers is simplified. In order of importance:
+    * automatic application of schema upgrades
+    * automatic creation of bootstrap repos
+    * Servers do not need to connect to SCC or any external repo
+    * Server/Hub GUI alert when updates are available for the Server/Hub itself
+    * installation of Servers is automated via Salt
+    * in principle any Server configuration can also be done via Salt (eg. creation of pillar stores, users, groups...)
+    * default DB backup via Taskomatic
+  * Hub implements UIs to act on Servers
+  * Optionally centralized definition of users and permissions
+
+### Non-functional requirements
+  * Performance: support high number of clients and Servers
+    * maximum expected number of clients: above 100k
+    * maximum expected number of Servers: 9k
+    * typical numbers:
+      * 3 to 20 Servers with a few thousand clients each ("large data center scenario")
+      * a few thousand Servers with 10-20 clients each ("large retailer scenario")
+  * Compatibility: we do not expect non-SUSE OSs soon, but design should accommodate them
+  * Maintainability: any new component should be designed in a modern way, with containerization/scalability/HA in mind
+  * Performance: Server functionality can stripped down in order to achieve smaller hardware footprint
+
+### Requirements explicitly excluded from this effort's scope
+Those are part of the long-term vision but were not given priority over other requirements, will not be analyzed further in the scope of this effort.
+
+  * Monitoring features. It's likely interesting but it's a totally separate research and implementation effort
+    * any "real-time" updates of data in the Hub. Hub will only host "high-latency", reporting data
+  * Provisioning features: Cobbler/Kickstart/AutoYAST
+  * Load balancing/high availability of Servers
+  * General backup of Servers (certificates, PKI files, Salt files, configuration files, rpms, own scripts...)
+  * SCAP auditing
+  * Subscription matching
+  * Centralized logging infrastructure
+
+## Long-term solution vision
+[Long-term solution vision]: #long-term-solution-vision
+
+* Employ several Servers - each Server managing a distinct subset of clients
```

Comment:
I would prefer leave that for a different set of RFCs, else even for a vision it would be too broad. We're always in time to add material with a new PR :wink: